javascript/urdf-viewer-element.js

```diff
@@ -405,6 +405,22 @@ class URDFViewer extends HTMLElement {
 
             let totalMeshes = 0;
             let meshesLoaded = 0;
+
+            if (pkg.includes(',')) {
```

Comment:
Should this check for inclusion of `:` and `,`? Or even just `:`? This will catch the case where someone provides two packages, but not one. Consider

`T12:.../urdf/T12, ATHLETE:.../urdf/ATHLETE`

vs

`T12:.../urdf/T12`

Reply:
Oh yes this is a good point!

Reply:
Done